peer/brontide.go

```diff
@@ -489,6 +508,16 @@ type Brontide struct {
 
 	// log is a peer-specific logging instance.
 	log btclog.Logger
+
+	// SendBackup indicates if to send back up to this field.
+	SendBackup int
+
+	// LatestPeerBackup is the latest backup we have stored with a peer.
+	LatestPeerBackup []byte
```

Comment:
I'm not clear on what this field's purpose is, or why it is necessary.

Reply:
An in-memory way to show the latest backup we have with the peer. It would also help with recovering the backed up data from the peer. As that field is assigned the data when we receive a `yourpeerstorage` message from the peer.